### PR TITLE
Update privacy policy, and put it on its own page

### LINKED ISSUE #68: Check and deploy amended Privacy Policy
https://docs.google.com/document/d/1kXHhPIMYzi19o8kYgpCi6Rfh-IcJZvqHNXe0IRbjxf0/edit# (only viewable by mySociety employees).

This expanded Privacy Policy includes the required clauses for incoming the GDPR regulations. Much of it is replicated from the Mapumental one. Again, @abimysoc might like to check for factualness.

### PR DESCRIPTION
**Pull request by zarino**
Fixes #68.

### CODE REVIEW

File: mapit_mysociety_org/templates/mapit/privacy.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+<section id="usage">
+<h2>How we use your data</h2>
+
+<p>We use this information only for the processing of your purchase, and company
+accounting — which may involve sharing it with our accountant or HMRC.</p>
+
+<p>We may, from time to time, send you important information about the service
+where changes or updates may be relevant to your use of the service.</p>
+
+<p>We do our utmost to store your data it securely. At any time you can request
```
Comment (dracos) [inline mapit_mysociety_org/templates/mapit/privacy.html]: Superfluous it here.

File: mapit_mysociety_org/templates/mapit/privacy.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+<section id="cookies">
+<h2>Cookies</h2>
+
+<p>This service uses a session cookie in order to know whether users are logged
+in and for the short term storage of information necessary to provide the
+service such as the volume of maps requested by the user. It also uses a CSRF
+(Cross Site Request Forgery) protection cookie, to prevent cross site request
+forgeries which should not store any personal information about the user.</p>
+
+<p>Like millions of other websites we use Google Analytics to help us
```
Comment (dracos) [inline mapit_mysociety_org/templates/mapit/privacy.html]: This should match the GA text used by all the other policies, please.

**Review by dracos — COMMENTED** [review]
One typo, one GA section change.

**Comment by zarino** [comment]
Thanks @dracos – all fixed now! Good to merge?

**Review by dracos — APPROVED** [review]

**Pull request merged**